samples/client/petstore/cpp-qt5/client/PFXHttpRequest.cpp

```diff
@@ -374,6 +374,7 @@ void PFXHttpRequestWorker::on_manager_finished(QNetworkReply *reply) {
             headers.insert(item.first, item.second);
         }
     }
+    disconnect(this, nullptr, nullptr, nullptr);
```

Comment:
Actually this change broke tests.

Reply:
We can wait for the build to finish. CI should fail.

Reply:
Yes it fails locally on my side...